linfa-supervised/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "linfa-supervised"
```

Comment:
I would probably prefer this to be named `linfa-linear` - we will have many more algorithm belonging to the family of supervised learning and we don't want to have all of them in the same sub-crate.